roles/servicetelemetry/tasks/component_events_smartgateway.yml

```diff
@@ -29,7 +29,7 @@
         useBasicAuth: true
         elasticUser: '{{ elastic_user }}'
         elasticPass: '{{ elastic_pass }}'
-        containerImagePath: 'quay.io/infrawatch/smart-gateway:latest'
+        containerImagePath: 'quay.io/infrawatch/smart-gateway:v1.0.0-beta1'
```

Comment:
Reconfirmed that this actually exists.

https://quay.io/repository/infrawatch/smart-gateway?tag=latest&tab=tags

Reply:
This is the last part I'm trying to figure out. I actually want to leave this at `latest` for development/release purposes and only adjust this when I go to do an actual release, with it being only adjusted based on the STF release you're going to deploy with.

So I think what this is going to mean is making this something that gets passed in by the default CR when deploying a `ServiceTelemetry` instance. Since the CSV contains the example CR, we could actually just provide a `smartgatewayContainerImageTag` value into the CR to override the default of `latest`.

Anyways, something else I'll be playing around with tomorrow as well.